databases/aws-rds-tls/src/main/java/gov/cloud/rdstlstest/config/DataSourceConfig.java

```diff
@@ -0,0 +1,42 @@
+package gov.cloud.rdstlstest.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+import javax.sql.DataSource;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.*;
+
+@Configuration
+public class DataSourceConfig {
```

Comment:
There's a better way to do this, through `@ConfigurationProperties`, but I can't remember the specifics. Not a blocker, mostly fyi.